Profiles/Menna/logistic_regression.py

```diff
@@ -0,0 +1,95 @@
+from data_reader.reader import CsvReader
+from util import *
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pdb
+
+#"""
+
+class LogisticRegression(object):
+
+    def __init__(self, learning_rate=0.01, epochs=50):
+        self.__epochs= epochs
+        self.__learning_rate = learning_rate
+
+    def fit(self, X, y):
+        print (self.w_)
+        return 0
```

Comment:
@mennatullah65 Well Done for completing the practical successfully, I have only one comment that line 17, 18 will produce error .. if you removed them everything will work as expected.